crates/turbopack-node/src/evaluate.rs

```diff
@@ -229,13 +285,76 @@ pub async fn evaluate(
     .await
     .map_err(|(e, _)| e)?;
 
+    // If we reach an End or Error value immediately, then we can return an easy
+    // Single response. If not, we need to stream multiple responses out.
+    let output = pull_operation(&mut operation, cwd, context_ident_for_issue).await?;
+    let data = match output {
+        LoopResult::End(data) => {
+            if kill {
+                operation.wait_or_kill().await?;
```

Comment:
Do we need to wait for the operation to be finished before returning here? If not, you can spawn this in a different task.
```suggestion
                tokio::spawn(async move {
                    let _ = operation.wait_or_kill().await;
                });
```